arch/arm/dts/mt-connect-u-boot.dtsi

```diff
@@ -0,0 +1,141 @@
+// SPDX-License-Identifier: GPL-2.0+
+/*
+ * Copyright 2019, 2021 NXP
```

Comment:
@smorris-witekio if this is a brand new file do we need to assign copyright to NXP? Do we need this at all or should this be MultiTracks,com?

Reply:
Oh, good point, yes, I will change that to assign copyright to multitracks. Do you have a specific string you would like me to use? "Copyright 2024 Multitracks.com"?

Reply:
That file was just originally based on the evk, and I forgot to change the string

Reply:
@smorris-witekio MultiTracks.com capital M capital T and you have it.

Reply:
https://github.com/MultiTracksDotCom/uboot-imx/pull/3/commits/f199770f2c092f02ec50556e609c146768228b50
Added to PR#3

Reply:
I could add it to this PR, it would just take more time. Just let me know if putting in PR3 is not okay